com.unity.cinemachine/Runtime/Deprecated/CinemachineInputProvider.cs

```diff
@@ -132,3 +132,25 @@ namespace Unity.Cinemachine
     public class CinemachineInputProvider : MonoBehaviour {}
 }
 #endif
+
+namespace Unity.Cinemachine
+{
+    [Obsolete("IInputAxisProvider is deprecated.  Use InputAxis and InputAxisController instead")]
```

Comment:
`using System` is missing for this. It is hidden by the #if

Reply:
Good catch, thanks for checking!